lib/src/style/Theme.cpp

```diff
@@ -175,44 +215,45 @@ std::optional<Theme> Theme::fromJsonDoc(const QJsonDocument& jsonDoc) {
 
 void Theme::initializeFonts() {
   // Fonts.
-  const auto defaultFont = QFont(QStringLiteral("Inter"));
-  const auto fixedFont = QFont(QStringLiteral("Roboto Mono"));
-  const auto dpi = QGuiApplication::primaryScreen()->logicalDotsPerInch();
+  const auto defaultFont = useSystemFont ? QFontDatabase::systemFont(QFontDatabase::GeneralFont) : QFont(QStringLiteral("Inter"));
+  const auto fixedFont = useSystemFont ? QFontDatabase::systemFont(QFontDatabase::FixedFont) : QFont(QStringLiteral("Roboto Mono"));
+  const auto titleFont = useSystemFont ? QFontDatabase::systemFont(QFontDatabase::TitleFont) : QFont(QStringLiteral("InterDisplay"));
+
   fontRegular = defaultFont;
   fontRegular.setWeight(QFont::Weight::Normal);
-  fontRegular.setPointSizeF(pixelSizeToPointSize(fontSize, dpi));
+  fontRegular.setPointSize(useSystemFont && !specifiedFontSize ? defaultFont.pointSize() : fontSize);
```

Comment:
Please use `setPointSizeF` instead of `setPointSize`

Reply:
Why? The fontSize variables are ints not floats.

Reply:
Because someday I would like to switch to qreal values, which is the standard.

Reply:
Would you like me to change the json values to be floats instead of ints?